server/src/entity/SoloMarsEventCard.ts

```diff
@@ -0,0 +1,18 @@
+import { Column, Entity, PrimaryGeneratedColumn, ManyToMany, JoinColumn} from "typeorm";
+import { SoloMarsEventDeck } from "@port-of-mars/server/entity/SoloMarsEventDeck";
+
```

Comment:
These warning annotations are from eslint and usually need to be manually resolved. Formatting can be done automatically most of the time  (`docker compose exec server yarn style:fix` or in vscode with the prettier plugin)